e2e/client-auth/generate.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
I suggest using the `${KEY_DIR}` similar troughout the script as the `${CERT_DIR}`